src/wp2hugo/cmd/wp2hugo/main.go

```diff
@@ -16,6 +16,7 @@ var (
 	sourceFile                     = flag.String("source", "", "file path to the source WordPress XML file")
 	outputDir                      = flag.String("output", "/tmp", "dir path to write the Hugo-generated data to")
 	downloadMedia                  = flag.Bool("download-media", false, "download media files embedded in the WordPress content")
+	downloadAll                    = flag.Bool("download-all", false, "download all media from WordPress library, whether used in content or not")
```

Comment:
why not more explicit `download-all-media`?

Reply:
To me it seems like `download-all-media` is another option on top of `download-media`, but `download-all` is independent

Reply:
> To me it seems like download-all-media is another option on top of download-media, but download-all is independent

Interesting, so, you are saying non-media attachments are covered too?
Then it makes sense that it is called `download-all`.